judgments/tests/tests.py

```diff
@@ -44,10 +43,11 @@ def test_judgment_not_published_404_response(self, mock_judgment):
         self.assertIn("Page not found", decoded_response)
         self.assertEqual(response.status_code, 404)
 
-    @patch("judgments.views.detail.get_judgment_by_uri")
-    def test_judgment_not_found_404_response(self, mock_judgment):
-        mock_judgment.side_effect = MarklogicResourceNotFoundError()
-
+    @patch(
+        "caselawclient.models.judgments.Judgment.judgment_exists",
```

Comment:
This feels like we're patching the wrong thing; we're not testing that the model doesn't the right thing if the judgment doesn't exist, we're testing that the view does the right thing if the model tells us the judgment doesn't exist.